### ISSUE TITLE
Contribution guide points out "Easy starter" issue despite no issues being labeled as such

### ISSUE THREAD

**Issue by chingucoding** [microsoft/terminal#14633]
Pretty much as the title says. In the [contribution documentation](https://github.com/microsoft/terminal/blob/main/CONTRIBUTING.md), issues labeled as "Easy starter" are linked, however no issues are labeled as such, and the label also doesn't exist. I think the documentation should be updated to not link to an issue query that returns nothing.

**Comment by zadjii-msft** [comment]
Good catch, I just went ahead and patched that in 21a62c5fefa36c619bd15bc80d43f54b4d878a5d. Thanks!
